controllers/admin/AdminFeaturesController.php

```diff
@@ -190,18 +190,20 @@ public function renderForm()
 
     public function initPageHeaderToolbar()
     {
-        if (empty($this->display)) {
-            $this->page_header_toolbar_btn['new_feature'] = array(
-                'href' => self::$currentIndex.'&addfeature&token='.$this->token,
-                'desc' => $this->trans('Add new feature', array(), 'Admin.Catalog.Feature'),
-                'icon' => 'process-icon-new'
-            );
+        if (Feature::isFeatureActive()) {
```

Comment:
It's not really the same : 
- sometimes $this->display == 'something', sometimes empty($this->display) 
- Combination:: vs Feature::